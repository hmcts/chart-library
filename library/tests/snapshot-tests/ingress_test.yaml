suite: Snapshot tests
templates:
  - templates/v2/ingress.yaml
tests:
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: {}