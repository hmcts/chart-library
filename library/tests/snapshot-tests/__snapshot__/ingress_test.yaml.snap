manifest should match snapshot:
  1: |
    apiVersion: networking.k8s.io/v1
    kind: Ingress
    metadata:
      annotations: null
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: release-name-library
        helm.sh/chart: library-0.1.0
      name: release-name-library
    spec:
      ingressClassName: traefik
      rules:
        - host: release-name.service.core-compute-preview.internal
          http:
            paths:
              - backend:
                  service:
                    name: release-name
                    port:
                      number: 80
                path: /test
                pathType: Prefix
              - backend:
                  service:
                    name: release-name-library
                    port:
                      number: 80
                path: /
                pathType: Prefix
        - host: release-name-ci-01.platform.hmcts.net
          http:
            paths:
              - backend:
                  service:
                    name: release-name
                    port:
                      number: 80
                path: /test
                pathType: Prefix
              - backend:
                  service:
                    name: release-name-library
                    port:
                      number: 80
                path: /
                pathType: Prefix
        - host: ci-02.platform.hmcts.net
          http:
            paths:
              - backend:
                  service:
                    name: release-name
                    port:
                      number: 80
                path: /test
                pathType: Prefix
              - backend:
                  service:
                    name: release-name-library
                    port:
                      number: 80
                path: /
                pathType: Prefix
